src/chatd.cpp

```diff
@@ -2947,6 +2947,16 @@ void Chat::attachmentHistDone()
     mAttachmentNodes->finishFetchingFromServer();
 }
 
+void Chat::requestUserAttributes(Id sender)
+{
+    if (!isGroup() && !isPublic())
```

Comment:
Not sure, but I think you want here:
```suggestion
    if (!(isGroup() && isPublic()))
```
If so, you can directly skip the `isGroup()` (all public chats are groupchats).
```suggestion
    if (!isPublic())
```